### ISSUE TITLE
New planet shows "eyre: failed tube from hark-update to json"

### ISSUE THREAD

**Issue by patosullivan** [urbit/urbit#3975]
**Describe the bug**
I have a ship I've spun up in just the past few days. Seemingly after today's OTA update it has started printing this output to the terminal over and over:

`eyre: failed tube from hark-update to json`

It also pegs CPU usage at 100%.

If I hit `ctrl-c` this goes away and CPU usage goes back to normal, but the following is printed to the console:

```
recover: dig: intr
crud: %request event failed
[%poke p=%request]
%leap
wire=/channel/subscription/1605830896031-753c69/84/~[my-shipname]/hark-store
  bar-stack
~[
  ~[//http-server/0v10.9psbh/214/38]
  ~[
    /e/channel/subscription/1605830896031-753c69/84/~[my-shipname]/hark-store
    //http-server/0v10.9psbh/214/38
  ]
  ~[//http-server/0v10.9psbh/214/38]
]
take: failed
/sys/vane/eyre/hoon:<[2.329 3].[2.503 5]>
/sys/vane/eyre/hoon:<[2.330 3].[2.503 5]>
/sys/vane/eyre/hoon:<[2.334 3].[2.503 5]>
/sys/vane/eyre/hoon:<[2.335 3].[2.503 5]>
/sys/vane/eyre/hoon:<[2.344 3].[2.503 5]>
/sys/vane/eyre/hoon:<[2.346 3].[2.503 5]>
/sys/vane/eyre/hoon:<[2.346 7].[2.356 9]>
/sys/vane/eyre/hoon:<[2.348 7].[2.356 9]>
/sys/vane/eyre/hoon:<[2.354 28].[2.354 35]>
/sys/vane/eyre/hoon:<[2.425 5].[2.464 7]>
/sys/vane/eyre/hoon:<[2.428 5].[2.464 7]>
/sys/vane/eyre/hoon:<[2.430 5].[2.464 7]>
/sys/vane/eyre/hoon:<[2.451 7].[2.463 31]>
/sys/vane/eyre/hoon:<[2.452 7].[2.463 31]>
[ i=~.channel
    t
  [i=%subscription t=[i=~.1605830896031-753c69 t=/84/~[my-shipname]/hark-store]]
]
/sys/vane/eyre/hoon:<[2.453 7].[2.463 31]>
/sys/vane/eyre/hoon:<[2.454 7].[2.463 31]>
/sys/vane/eyre/hoon:<[2.455 7].[2.463 31]>
/sys/vane/eyre/hoon:<[2.456 7].[2.463 31]>
/sys/vane/eyre/hoon:<[2.457 7].[2.463 31]>
/sys/vane/eyre/hoon:<[2.460 7].[2.463 31]>
/sys/vane/eyre/hoon:<[2.461 9].[2.462 61]>
/sys/vane/eyre/hoon:<[1.463 7].[1.487 60]>
/sys/vane/eyre/hoon:<[1.471 7].[1.487 60]>
/sys/vane/eyre/hoon:<[1.475 9].[1.475 48]>
/sys/vane/eyre/hoon:<[1.500 7].[1.611 9]>
/sys/vane/eyre/hoon:<[1.502 7].[1.611 9]>
/sys/vane/eyre/hoon:<[1.504 7].[1.611 9]>
/sys/vane/eyre/hoon:<[1.513 7].[1.611 9]>
/sys/vane/eyre/hoon:<[1.521 7].[1.611 9]>
/sys/vane/eyre/hoon:<[1.522 9].[1.522 39]>
/sys/vane/eyre/hoon:<[1.641 7].[1.699 9]>
/sys/vane/eyre/hoon:<[1.644 7].[1.699 9]>
/sys/vane/eyre/hoon:<[1.645 9].[1.662 53]>
/sys/vane/eyre/hoon:<[1.646 9].[1.662 53]>
/sys/vane/eyre/hoon:<[1.649 9].[1.662 53]>
/sys/vane/eyre/hoon:<[1.650 9].[1.662 53]>
/sys/vane/eyre/hoon:<[1.651 9].[1.662 53]>
/sys/vane/eyre/hoon:<[1.656 9].[1.662 53]>
/sys/vane/eyre/hoon:<[1.659 9].[1.662 53]>
/sys/vane/eyre/hoon:<[1.659 18].[1.659 49]>
```

**System (please supply the following information, if relevant):**
 - OS: linux64
 - Vere and Urbit OS versions: urbit 10.8. Not sure how to check the vere version.
 - Your ship's `%base` hash (use `+trouble` to check): 0v7.t16i9.avhp4.pflq0.1c79d.rplo0.ho0i2.9ovj1.kddvj.e1snn.0kb4n

**Additional context**
If I refresh the landscape the problem will come back.

**Comment by Sneagan** [comment]
I have the same thing on a new planet spun up in the last couple of days. Here's my own `+trouble` output:

```
> +trouble
[ [%base-hash ~[0ve.0bigv.s2vc9.qilo1.b4hol.cf9ug.e8ubn.eu0k2.0h1s2.6oue8.4vq4g]]
  [%sour-hash 0ve.0bigv.s2vc9.qilo1.b4hol.cf9ug.e8ubn.eu0k2.0h1s2.6oue8.4vq4g]
  [%home-hash 0ve.0bigv.s2vc9.qilo1.b4hol.cf9ug.e8ubn.eu0k2.0h1s2.6oue8.4vq4g]
  [%kids-hash 0v1f.q72m8.4rsom.u7qh5.6m3so.n21fe.j7l94.u3il3.s24pt.vgob2.386ur]
  [%glob-hash 0v4.fpa4r.s6dtc.h8tps.62jv0.qn0fj %done]
  [%our ship=~darduc-mitfen point='2220194863' life=[~ 1] rift=[~ 0]]
  [%sponsor ship=~nisfeb point='31791' life=[~ 1] rift=[~ 0]]
  [%dopzod ship=~dopzod point='4608' life=[~ 3] rift=[~ 2]]
  "Compare lifes and rifts to values here:"
  "https://etherscan.io/address/azimuth.eth#readContract"
  "  life - getKeyRevisionNumber"
  "  rift - getContinuityNumber"
  ~
]
```

**Comment by vvisigoth** [comment]
@philipcmonk @Fang- can you take a look at this?

**Comment by Fang-** [comment]
> `eyre: failed tube from hark-update to json`

The message here just means Eyre attempted and failed to convert a hark-update to json. The CPU usage probably comes from it trying this very often, for many updates/notifications. Ctrl-C-ing interrupts the attempts, but refreshing landscape opens a new subscriptions, sending this info all over again.

Unclear why this would crash though. The mark itself just calls out to the library:

https://github.com/urbit/urbit/blob/a9d6811299fc520e4f46f4abdc626de536261a75/pkg/arvo/mar/hark/update.hoon#L7-L9

And [the library](https://github.com/urbit/urbit/blob/a9d6811299fc520e4f46f4abdc626de536261a75/pkg/arvo/lib/hark/store.hoon#L72) has a lot of logic for this, but it all looks like straightforward json conversion. Nothing jumps out to me as "could crash". @liam-fitzgerald any risky edge cases that come to mind?

Maybe Eyre should print the stack trace in such cases? Would be even noisier, but might give a better indication of what's actually going wrong.

**Comment by patosullivan** [comment]
Having the problem reoccur every time I refresh the landscape is becoming an issue. I'm guessing the only way to fix it might be breaching and starting over? If there's anything else I should try, let me know.

**Comment by liam-fitzgerald** [comment]
I can't find @Sneagan 's bash hash anywhere in recent files, so maybe need to force an update? @patosullivan could I get you to confirm that running `-build-file %/mar/hark/update/hoon` does not print a stack trace?

**Comment by patosullivan** [comment]
@liam-fitzgerald it does print this:

```
> -build-file %/mar/hark/update/hoon< 3.vkg [ upd ^#16 ?( [ %added time=@da index ?([%chat chat=/ mention=?(%.y %.n)] [%graph group=[entity=@p name=@tas] graph=[entity=@p name=@tas] module=@t description=@t] [%group group=[entity=@p name=@tas] description=@t]) notification [ date=@da read=?(%.y %.n) contents ?( [ %chat list it( [ uid=@uvH number=@ author=@p when=@da letter ?( [%code expression=@t output=it(?([%leaf p=""] [%palm p=[p="" q="" r="" s=""] q=#6] [%plum p=#7] [%rose p=[p="" q="" r=""] q=#6]))] [%me narrative=@t] [%text text=@t] [%url url=@t] ) ] ) ] [ %graph list it( [ author=@p index=it(@) time-sent=@da contents it( ?( [%code expression=@t output=it(?([%leaf p=""] [%palm p=[p="" q="" r="" s=""] q=#6] [%plum p=#7] [%rose p=[p="" q="" r=""] q=#6]))] [%mention ship=@p] [%reference uid=[resource=[entity=@p name=@tas] index=it(@)]] [%text text=@t] [%url url=@t] ) ) hash=u(@ux) signatures=nlr([p=@ux q=@p r=@ud]) ] ) ] [ %group list it( ?( ?([%add-members resource=[entity=@p name=@tas] ships=nlr(@p)] [%remove-members resource=[entity=@p name=@tas] ships=nlr(@p)]) ?( [%add group-path=/ resource=[app-name=@tas app-path=/] metadata=[title=@t description=@t color=@ux date-created=@da creator=@p module=@tas]] [%remove group-path=/ resource=[app-name=@tas app-path=/]] ) ) ) ] ) ] ] [%count count=@ud] [%more more=it(#16)] [ %timebox time=@da archived=?(%.y %.n) list it( [ ?([%chat chat=/ mention=?(%.y %.n)] [%graph group=[entity=@p name=@tas] graph=[entity=@p name=@tas] module=@t description=@t] [%group group=[entity=@p name=@tas] description=@t]) date=@da read=?(%.y %.n) contents ?( [ %chat list it( [ uid=@uvH number=@ author=@p when=@da letter ?( [%code expression=@t output=it(?([%leaf p=""] [%palm p=[p="" q="" r="" s=""] q=#6] [%plum p=#7] [%rose p=[p="" q="" r=""] q=#6]))] [%me narrative=@t] [%text text=@t] [%url url=@t] ) ] ) ] [ %graph list it( [ author=@p index=it(@) time-sent=@da contents it( ?( [%code expression=@t output=it(?([%leaf p=""] [%palm p=[p="" q="" r="" s=""] q=#6] [%plum p=#7] [%rose p=[p="" q="" r=""] q=#6]))] [%mention ship=@p] [%reference uid=[resource=[entity=@p name=@tas] index=it(@)]] [%text text=@t] [%url url=@t] ) ) hash=u(@ux) signatures=nlr([p=@ux q=@p r=@ud]) ] ) ] [ %group list it( ?( ?([%add-members resource=[entity=@p name=@tas] ships=nlr(@p)] [%remove-members resource=[entity=@p name=@tas] ships=nlr(@p)]) ?( [%add group-path=/ resource=[app-name=@tas app-path=/] metadata=[title=@t description=@t color=@ux date-created=@da creator=@p module=@tas]] [%remove group-path=/ resource=[app-name=@tas app-path=/]] ) ) ) ] ) ] ) ] [ %unreads unreads it( [ ?([%chat chat=/ mention=?(%.y %.n)] [%graph group=[entity=@p name=@tas] graph=[entity=@p name=@tas] module=@t description=@t] [%group group=[entity=@p name=@tas] description=@t]) @ud ] ) ] ?( [ %add index ?([%chat chat=/ mention=?(%.y %.n)] [%graph group=[entity=@p name=@tas] graph=[entity=@p name=@tas] module=@t description=@t] [%group group=[entity=@p name=@tas] description=@t]) notification [ date=@da read=?(%.y %.n) contents ?( [ %chat list it( [ uid=@uvH number=@ author=@p when=@da letter ?( [%code expression=@t output=it(?([%leaf p=""] [%palm p=[p="" q="" r="" s=""] q=#6] [%plum p=#7] [%rose p=[p="" q="" r=""] q=#6]))] [%me narrative=@t] [%text text=@t] [%url url=@t] ) ] ) ] [ %graph list it( [ author=@p index=it(@) time-sent=@da contents it( ?( [ %code expression=@t output it( ?( [%leaf p=""] [%palm p=[p="" q="" r="" s=""] q=#6] [%plum p=#7] [ %rose p=[p="" q="" r=""] q ^#6 ?( %~ [ i ?( [%leaf p=""] [%palm p=[p="" q="" r="" s=""] q=#6] [ %plum p ^#7 ?( [%para prefix=@t lines=*''] [%sbrk kid=#7] [%tree fmt=[wide=u([delimit=@t enclose=u([p=@t q=@t])]) tall=u([intro=@t indef=u([sigil=@t final=@t])])] kids=it(#7)] @t ) ] [%rose p=[p="" q="" r=""] q=#6] ) t=#6 ] ) ] ) ) ] [%mention ship=@p] [%reference uid=[resource=[entity=@p name=@tas] index=it(@)]] [%text text=@t] [%url url=@t] ) ) hash=u(@ux) signatures=nlr([p=@ux q=@p r=@ud]) ] ) ] [ %group list it( ?( ?([%add-members resource=[entity=@p name=@tas] ships=nlr(@p)] [%remove-members resource=[entity=@p name=@tas] ships=nlr(@p)]) ?( [%add group-path=/ resource=[app-name=@tas app-path=/] metadata=[title=@t description=@t color=@ux date-created=@da creator=@p module=@tas]] [%remove group-path=/ resource=[app-name=@tas app-path=/]] ) ) ) ] ) ] ] [ %archive time=@da ?([%chat chat=/ mention=?(%.y %.n)] [%graph group=[entity=@p name=@tas] graph=[entity=@p name=@tas] module=@t description=@t] [%group group=[entity=@p name=@tas] description=@t]) ] [ %read time=@da ?([%chat chat=/ mention=?(%.y %.n)] [%graph group=[entity=@p name=@tas] graph=[entity=@p name=@tas] module=@t description=@t] [%group group=[entity=@p name=@tas] description=@t]) ] [ %read-index ?([%chat chat=/ mention=?(%.y %.n)] [%graph group=[entity=@p name=@tas] graph=[entity=@p name=@tas] module=@t description=@t] [%group group=[entity=@p name=@tas] description=@t]) ] [%seen %~] [%set-dnd dnd=?(%.y %.n)] [ %unread time=@da ?([%chat chat=/ mention=?(%.y %.n)] [%graph group=[entity=@p name=@tas] graph=[entity=@p name=@tas] module=@t description=@t] [%group group=[entity=@p name=@tas] description=@t]) ] ) ) [ < 2?msv [ chat-store [ <9?awd 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <3?dxd [sur=<9?awd 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> ] group-store [ < 5?ach [ resource < 1.vpn [ * < 8.fec [sur=<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>] > ] > sur < 4?nml [ <3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <7?gwa [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > <7?gwa [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > < 4?nml [ <3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <7?gwa [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > ] graph-store [ < 13.rhp [ <9.zoj [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > <9.zoj [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> < 8.ygc [ res < 1.vpn [ * < 8.fec [sur=<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>] > ] > pos=<9.zoj [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> sur < 13.rhp [ <9.zoj [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > ] resource < 1.vpn [* <8.fec [sur=<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]>] > post=<9.zoj [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> sur < 9?lxc [ chat-store=<9?awd 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> metadata-store=<9.kmx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> group-store < 4?nml [ <3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <7?gwa [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > post=<9.zoj [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> graph-store < 13.rhp [ <9.zoj [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > <3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > < 9?lxc [ chat-store=<9?awd 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> metadata-store=<9.kmx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> group-store < 4?nml [ <3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <7?gwa [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > post=<9.zoj [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> graph-store < 13.rhp [ <9.zoj [<3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141>]> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > <3?hzx 20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ] > ] <20.hmj 31.lta 9.ueb 36.oqd 94.cfh 258.ltk 51.pkf 129.pti 43.hoc 1.ane %141> ]>


```

**Comment by patosullivan** [comment]
Sorry,  ran that command from the terminal within the web app on my phone, looks like the formatting didn't come along when I pasted it. I'll run the command again on a real terminal tomorrow.

**Comment by liam-fitzgerald** [comment]
@patosullivan could you try running `:goad %force` and see if that fixes it? Failing that, could you run the following and post the output of the last two? It might contain some personal stuff, feel free to redact anything you don't want publically. Alternatively, you could just email it to me  ( liam.fitzgerald@tlon.io )
`=hark -build-file %/lib/hark/store/hoon`
`.^(update:hark %gx /(scot %p our)/hark-store/(scot %da now)/recent/inbox/0/3/noun`
`.^(update:hark %gx /(scot %p our)/hark-store/(scot %da now)/recent/archive/0/3/noun`

**Comment by patosullivan** [comment]
`:goad %force` did it, thanks! Would you mind explaining what that command does?

**Comment by liam-fitzgerald** [comment]
It reloads all of your gall agents (your apps). `hark-store` (the notifications store) keeps a resource-indexed cache of notifications to improve query times as the core datastructure is time-indexed. This cache is rebuilt when we reload the app, so we can just migrate the main datastructure and then rebuild the cache, instead of having to increase the complexity of migration logic. So `:goad %force` will ensure that hark rebuilds its cache.

This will be fixed by #4016  which addresses the cache falling out of sync with the core datastructure under certain conditions.

**Comment by Sneagan** [comment]
This issue is no longer happening for me since the latest OTA. I have other issues now but I consider this one resolved. Thanks, all!
